.travis.yml

```diff
@@ -46,4 +46,5 @@ script:
   - make docker-bootstrap
   - docker-compose up -d
   - sleep 30
+  - make load-synthetic-cclf-data
```

Comment:
Previously, loading the data was part of `make docker-bootstrap`, so it was not a step required by folks to run manually in their local setup.  Now, it is an explicitly separate target, which would have to be run separately in the local env setup.  
We'll need to either (1) update the[ readme to specify the order of operations required ](https://github.com/CMSgov/bcda-app/blob/master/README.md#build--start)to get a local env setup, or (2) setup up `docker-boostrap` to bring up the compose stack and load the synthetic data so that user's don't have to invoke those targets separately.